File: Program's_Contributed_By_Contributors/C++ Programs/Array/Reversearray.cpp
/*Program to reverse the elements of array*/

#include<iostream>
using namespace std;
int main(){
    int n,temp;
    cout<<"Enter the size of array:";
    cin>>n;
    int array[n];
    cout<<"Enter the elements of array:";
    for(int i=0;i<n;i++){
        cin>>array[i];
    }
    for(int i=0,j=n-1;i<n/2;i++,j--){
        temp=array[i];
        array[i]=array[j];
        array[j]=temp;
    }
cout<<"Reversed elements of array is:";
    
    for(int i=0;i<n;i++){
        cout<<array[i]<<" ";
    }
    return 0;
}

File: Program's_Contributed_By_Contributors/Java_Programs/Array/ReverseArray.java
public class ReverseArray {
    public static void reverse(int numbers[]){
        int start = 0;
        int end = numbers.length-1;
        while(start<end){
            int temp = numbers[start];
            numbers[start] = numbers[end];
            numbers[end] = temp;
            start++;
            end--;
        }
    }
    public static void main(String args[]){
        int numbers[] = {1,2,3,4,5,6,7,8,9};
        reverse(numbers);
        for(int i=0; i<numbers.length;i++){
            System.out.print(numbers[i]+" ");
        }
    }    
}
